q2_demux/_summarize/assets/quality-plot.html

```diff
@@ -63,8 +63,23 @@ <h5 class="text-center">Forward Reads</h5>
   <div id="reverseContainer" class="col-xs-12 col-md-6">
     <h5 class="text-center">Reverse Reads</h5>
   </div>
+  <div class="row">
+    <div class="col-lg-12">
+      <a href="forward-index-quality-scores.csv">Download forward quality as CSV</a>
+    </div>
+  </div>
+  <div class="row">
+      <div class="col-lg-12">
+        <a href="reverse-index-quality-scores.csv'">Download reverse counts as CSV</a>
+      </div>
+  </div>
   {% else %}
   <div id="forwardContainer" class="col-xs-12 col-md-6 col-md-offset-3"></div>
+  <div class="row">
```

Comment:
Can you move this block ([L78](https://github.com/qiime2/q2-demux/pull/77/files#diff-fba5da0182f7db93468361264d2b2308R78) - [L82](https://github.com/qiime2/q2-demux/pull/77/files#diff-fba5da0182f7db93468361264d2b2308R82)) up into the `forwardContainer` `div` [above](https://github.com/qiime2/q2-demux/pull/77/files#diff-fba5da0182f7db93468361264d2b2308R77)?